README.md

```diff
@@ -1,16 +1,27 @@
 # Interview
 
-We will be creating a calendar view that displays streak information for a given user. 
+The intent of this interview is to:
+- be an exercise that lets us get a better idea of how you think and work. We want to see how you adapt to a new code base and solve the challenges we've left for you along the way.
+- be a reasonable simulation of really working together. At least as close as we can accomplish in a short time period
```

Comment:
Seeing as you end all the rest of the points with periods, can we for these two too?